sway/commands/move.c

```diff
@@ -127,8 +127,11 @@ static void container_move_to_container_from_direction(
 			if (destination->parent) {
 				container_insert_child(destination->parent, container, index);
 			} else {
-				workspace_insert_tiling(destination->workspace,
-						container, index);
+				struct sway_workspace *ws = destination->workspace;
```

Comment:
This branch needed changing because if we had `T[focused unfocused]` and ran `move left`, we'd generate `H[H[focused] T[unfocused]]`. Now we generate `H[T[focused] T[unfocused]]`. The other split change covers the `move right` case.